lib/msal-core/tests/mochaTests/error/AuthError.spec.ts

```diff
@@ -0,0 +1,50 @@
+import * as chai from "chai";
+import { AuthError, AuthErrorMessage } from "../../../src/error/AuthError";
+
+const expect = chai.expect;
+chai.config.includeStack = false;
```

Comment:
can this config be set in a more centralplace?

Reply:
Yes. Gearing towards this. +1

Reply:
I have centralized chai configuration in the chai.conf.js file